src/components/DropUncle.tsx

```diff
@@ -32,7 +47,7 @@ const DropUncle = ({ depth, path, simplePath }: { depth?: number; path: Path; si
       ref={dropTarget}
       style={{
         backgroundColor: testFlags.simulateDrop ? '#52305f' : undefined, // eggplant
-        height: '1.9em',
+        height: dragInProgress ? `${1.9 + dynamicHeight}em` : '1.9em',
```

Comment:
Is there a reason that `dragInProgress` is checked? I think it may work without it. `drop-end` is position: absolute so the height will not affect the layout.